Firmata.cpp

```diff
@@ -398,7 +437,11 @@ void FirmataClass::attach(uint8_t command, systemCallbackFunction newFunction)
  */
 void FirmataClass::attach(uint8_t command, stringCallbackFunction newFunction)
 {
-    parser.attach(command, (stringCallbackFunction)newFunction);
+  switch (command) {
```

Comment:
I can't imagine a more performant branch than an `if`, but I was assuming they would distill down into the same assembly; I will check into it and get back. I just wanted to keep the same semantics, but I wouldn't sacrifice performance for it.